vsintegration/src/FSharp.UIResources/LanguageServicePerformanceOptionControl.xaml

```diff
@@ -19,10 +19,6 @@
             <StackPanel>
                 <GroupBox Header="{x:Static local:Strings.Project_Performance}">
                     <StackPanel>
-                        <CheckBox x:Name="enableInMemoryCrossProjectReferences"
-                                  IsChecked="{Binding EnableInMemoryCrossProjectReferences}"
-                                  Content="{x:Static local:Strings.Enable_in_memory_cross_project_references}"
-                                  ToolTip="{x:Static local:Strings.Tooltip_in_memory_cross_project_references}"/>
```

Comment:
We should probably also remove these strings from `Strings.resx` file.

Reply:
Oh, I must've missed, was wondering why translation baseline wasn't updated